cmd/main.go

```diff
@@ -339,7 +339,6 @@ var appCmds = []cli.Command{
 	diffCmd,
 	rmCmd,
 	eventCmd,
-	ilmCmd,
```

Comment:
If you want backward compatibility, it is enough to keep this varaible with hidden variable set to true in cli.Command{}

Reply:
No backward compatibility needed, we are breaking willfully @vadmeste